UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -24,7 +24,10 @@ public enum EventIds
         ESSPostProductIdentifiersRequestStart = 805017,
         ESSPostProductIdentifiersRequestCompleted = 805018,        
         ESSPostProductVersionsRequestStart = 805019,
-        ESSPostProductVersionsRequestCompleted = 805020        
+        ESSPostProductVersionsRequestCompleted = 805020,
+        SCSResponseStoreRequestStart = 805021,
+        SCSResponseStoreCompleted = 805022,
```

Comment:
Change SCSResponseStoreCompleted to SCSResponseStoreRequestCompleted

Reply:
Done